drivers/spi/spi_sf32lb.c

```diff
@@ -158,6 +158,14 @@ static int spi_sf32lb_configure(const struct device *dev, const struct spi_confi
 		top_ctrl &= ~SPI_TOP_CTRL_TTE;
 	}
 
+	if (config->operation & SPI_HOLD_ON_CS) {
```

Comment:
9f3875d47ee14332c637a0f843cb83eff6f994e8 has very few details on what is fixed... please add details/explain